### PR TITLE
Added describe pvc command

### PR DESCRIPTION
**Pull request by kkg-portworx**

### CODE REVIEW

File: cmd/describePvc.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	"github.com/spf13/cobra"
+)
+
+var describePvcCmd *cobra.Command
+
+var _ = RegisterCommandVar(func() {
+	describePvcCmd = &cobra.Command{
+		Use:   "pvc",
+		Short: "Describe Portworx volume for Kubernetes PVCs",
+		Long:  "Show detailed information of Portworx volume for Kuberntes PVCs",
```
Comment (siva-portworx) [inline cmd/describePvc.go]: Typo: Kuberntes -> Kubernetes

File: cmd/describePvc.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  This describes all pvcs that are Portworx volumes
+$ px describe pvc abc
+  This describes pvc abc
+$ px describe pvc abc xyz
+  This describes pvcs abc and xyz`,
+		RunE: describePvcExec,
+	}
+})
+
+// describeVolumeCmd represents the describeVolume command
```
Comment (siva-portworx) [inline cmd/describePvc.go]: Typo: describeVolumeCmd --> describePvcCmd

File: cmd/describePvc.go (hunk 1 above)
Comment (kkg-portworx) [reply cmd/describePvc.go]: Done

File: cmd/describePvc.go (hunk 2 above)
Comment (kkg-portworx) [reply cmd/describePvc.go]: done

File: cmd/getPvc.go [hunk 3]
```diff
@@ -66,27 +73,59 @@ func getPvcExec(cmd *cobra.Command, args []string) error {
...
+	foundNames := make([]string, 0, len(pvcNames))
+	for _, pxp := range pxpvcs {
+		if util.ListContains(pvcNames, pxp.Name) {
+			filtered = append(filtered, pxp)
+			foundNames = append(foundNames, pxp.Name)
+		}
+	}
+	if len(pvcNames) != len(filtered) {
+		for _, pxn := range pvcNames {
+			if util.ListContains(foundNames, pxn) == false {
```
Comment (lpabon) [inline cmd/getPvc.go]: For next time: can be done using `!util.ListContainer(..)`

**Comment by lpabon** [comment]
Tested:

**Pull request merged**